src/main/java/io/kestra/plugin/ai/completion/JSONStructuredExtraction.java

```diff
@@ -106,9 +117,12 @@
 )
 public class JSONStructuredExtraction extends Task implements RunnableTask<JSONStructuredExtraction.Output> {
 
-    @Schema(title = "Text prompt", description = "The input prompt for the AI model")
+    @Schema(
+        title = "Chat Messages",
+        description = "The list of chat messages for the extraction context. There can be only one system message, and the last message must be a user message."
+    )
     @NotNull
-    private Property<String> prompt;
```

Comment:
Please keep a single prompt.
The idea was to use a system message under the cover to add the structured instruction instead of concatenating them inside the user prompt.